Lib/test/test_ast/test_ast.py

```diff
@@ -2259,7 +2259,7 @@ def test_validation(self):
                          "got an invalid type in Constant: list")
 
     def test_singletons(self):
-        for const in (None, False, True, Ellipsis, b'', frozenset()):
+        for const in (None, False, True, Ellipsis, b''):
```

Comment:
Although `frozenset()` is not a singleton, we are losing some test coverage here in the default build. Would it be worth omitting `frozenset()` only in free-threaded builds?

Reply:
I don't think we're losing meaningful test coverage. The `test_values` test below includes a `compile_constant()` call with a `frozenset`. It uses `assertEquals` instead of `assertIs`, which I think is the appropriate comparison for non-singletons. I think *identity* preserving behavior of an empty `frozenset()` is incidental and not worth testing.

Reply:
I agree with @colesbury. Presense of `frozenset()` here seems incidental.